assets/svg/logo.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="70" height="70" viewBox="0 0 70 70" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M61.0312 70H9.07812C4.04688 70 0 65.9531 0 61.0312V9.07812C0 4.04688 4.04688 0 9.07812 0H61.0312C65.9531 0 70 4.04688 70 9.07812V61.0312C70 65.9531 65.9531 70 61.0312 70Z" fill="white"/>
+<path d="M47.2654 26.641C48.1029 25.6987 48.5217 24.5471 48.5217 23.2908C48.5217 20.5688 46.3231 18.2656 43.4964 18.2656H22.872C21.7204 18.2656 20.7782 18.6843 20.0453 19.4172C19.3125 20.15 18.8937 21.0923 18.8937 22.2439C18.8937 24.4424 20.6735 26.2222 22.872 26.2222H26.5363C29.9911 26.2222 31.7709 30.3052 29.4677 32.9225L19.5219 44.1246C18.6843 45.0668 18.2656 46.2185 18.2656 47.4748C18.2656 50.1968 20.4641 52.5 23.2908 52.5H42.4495C41.612 51.4531 41.0885 49.9874 41.0885 48.5217C41.0885 47.056 41.612 45.5903 42.4495 44.5434H40.3557C36.9008 44.5434 35.0164 40.4604 37.3196 37.8431L47.2654 26.641Z" fill="#327FEB"/>
+<path d="M48.5217 52.5C50.7188 52.5 52.5 50.7189 52.5 48.5217C52.5 46.3245 50.7188 44.5434 48.5217 44.5434C46.3245 44.5434 44.5434 46.3245 44.5434 48.5217C44.5434 50.7189 46.3245 52.5 48.5217 52.5Z" fill="#327FEB"/>
```

Comment:
can you optimize this SVG?
Using [SVGCleaner](https://github.com/RazrFalcon/svgcleaner)

Reply:
Yes Sir. I have optimized SVG using the SVGCleaner. 

![SVG Cleaner 7_26_2021 8_28_09 PM](https://user-images.githubusercontent.com/53579386/127011559-72d909ab-bdd9-4ed6-a400-d5f1a0c6ae19.png)